app/src/main/java/sistemas/puc/com/finantialapp/data/FinantialContract.java

```diff
@@ -0,0 +1,87 @@
+package sistemas.puc.com.finantialapp.data;
+
+import android.content.ContentResolver;
+import android.content.ContentUris;
+import android.net.Uri;
+import android.provider.BaseColumns;
+import android.text.format.Time;
+
+public class FinantialContract {
+
+    public static final String CONTENT_AUTHORITY = "sistemas.puc.com.finantialapp.app";
+
+    public static final Uri BASE_CONTENT_URI = Uri.parse("content://" + CONTENT_AUTHORITY);
+
+    public static final String PATH_MOEDA   = "moeda";
+    public static final String PATH_TESOURO = "tesouro";
+    public static final String PATH_INDICE  = "indice";
+
+    // TODO: Find a replacement for Time. Time is deprecated
+    public static long normalizeDate(long startDate) {
+        Time time = new Time();
+        time.set(startDate);
+        int julianDay = Time.getJulianDay(startDate, time.gmtoff);
+        return time.setJulianDay(julianDay);
+    }
+
+    public static final class MoedaEntry implements BaseColumns {
+
+        public static final Uri CONTENT_URI =
+                BASE_CONTENT_URI.buildUpon().appendPath(PATH_MOEDA).build();
+
+        public static final String CONTENT_TYPE =
+                ContentResolver.CURSOR_DIR_BASE_TYPE + "/" + CONTENT_AUTHORITY + "/" + PATH_MOEDA;
+        public static final String CONTENT_ITEM_TYPE =
+                ContentResolver.CURSOR_ITEM_BASE_TYPE + "/" + CONTENT_AUTHORITY + "/" + PATH_MOEDA;
+
+        public static final String TABLE_NAME = "moeda";
+
+        // TODO: Add moeda columns definitions
+        public static final String COLUMN_MOEDA_INITIALS  = "moeda_initials";
```

Comment:
`public static final String COLUMN_MOEDA_INITIALS  = "moeda_initials";`
The "initials" are called country code. https://en.wikipedia.org/wiki/ISO_4217
Maybe rename it to COLUMN_MOEDA_CODE ?

Reply:
I liked your idea. I was looking for a better name. Thank you.